test_vm/tests/suite/prove_commit3_test.rs

```diff
@@ -2,7 +2,7 @@ use fil_actors_integration_tests::tests::prove_commit_sectors2_test;
 use fil_actors_runtime::test_blockstores::MemoryBlockstore;
 use test_vm::TestVM;
 
-#[test]
+#[test_log::test]
```

Comment:
I assume you will revert the test_log usage and dependencies when preparing for merge. If we want to use test_log, I think that's a separate consideration that we should execute in `main` ahead of this PR, and uniformly across all tests. Feel free to start an issue/discussion about doing that.